app/components/pages/examples/form/base.js

```diff
@@ -0,0 +1,58 @@
+import {Form, Input, Select, Option, Error, SubmitButton, TextArea} from '/components/elements/form'
+
+export default ({onSubmit, validations}) =>
+  <Form name='exampleForm' onSubmit={onSubmit} validations={validations}>
+    <div>
+      <label for='fullName' >Full Name:</label>
```

Comment:
On Qee we had Fields that would render Label, Input and Error, and custom hints with one Component. I think it'd be nice to keep that to reduce boilerplate when rendering forms.